packages/app-page-builder-elements/src/renderers/cell.tsx

```diff
@@ -21,7 +21,7 @@ export const createCell = () => {
                     return styles;
                 }
 
-                styles.width = `${(size / 12) * 100}%`;
+                styles.width = `calc(${(size / 12) * 100}% - var(--cellWidthOffset, 0px))`;
```

Comment:
@adrians5j 
just in case, please check if you are looking on latest changes...
I see here an "outdated" tag for this file, and latest implementation is little bit different

Reply:
@adrians5j 
`cellWidthOffset` is used to define smth like a CSS grid gap, so cells have a spacing between them (that is configurable via UI).
The idea is that we need to reduce columns for few pixels to make a place for a gap between them.
So total width of the row is `=100%` (and not is `=100% spited between all cells + gap size * number of gaps`)

Reply:
I understand the part related to ensuring spacing between them, and I think it makes sense.

The thing that I don't understand is the use of CSS variable `--cellWidthOffset`. No CSS variables should be used in process of calculating element styles. It's all JS / objects based.

Ultimately, this CSS var should not be here. The width offset should be pulled from element data, the same way we're already doing for, for example, `size` (`const size = element.data?.settings?.grid?.size;`).